README.rst

```diff
@@ -15,9 +15,20 @@ Munin Node for Windows AKA munin-node-win32, is a Windows client for the Munin m
 It is written in C++ with most plugins built into the executable. 
 This is different from the standard munin-node client, which only uses external plugins written as shell and Perl scripts.
 
+Installer Usage
+------------------
+
+The installer may be run with different command-line flags.
+
+  * /HELP or /? Shows the help message.
+  * /SILENT or /VERYSILENT Instructs the installer to be silent or very silent.
+  * more options are shown in the /? help.
+
 Command Line Usage
 ------------------
 
+The munin-node executable takes these command-line flags.
```

Comment:
The purpose is to clarify without a doubt this is the munin-node executable, in contrast to the installer, since they both have command-line options.   Only saying "Command Line Usage" does not communicate that. 
The text mirrors the earlier installer section which has a title "Installer Usage", and then an explanatory sentence.   Here, another title, followed by another explanatory sentence, so they are similar.